src/Microsoft.ML/PredictionModel.cs

```diff
@@ -29,6 +29,33 @@ internal Runtime.EntryPoints.TransformModel PredictorModel
             get { return _predictorModel; }
         }
 
+        /// <summary>
+        /// Returns labels that correspond to indices of the score array in the case of 
+        /// multi-class classification problem.
+        /// </summary>
+        /// <param name="mapping">Label to score mapping</param>
+        /// <param name="scoreColumnName">Name of the score column</param>
+        /// <returns></returns>
+        public bool TryGetScoreLabelMapping(out string[] mapping, string scoreColumnName = "Score")
+        {
+            mapping = null;
+            IDataView idv = _predictorModel.Schema;
+            int colIndex = -1;
+            if (!idv.Schema.TryGetColumnIndex(scoreColumnName, out colIndex))
+                return false;
+
+            VBuffer<DvText> labels = default(VBuffer<DvText>);
+            idv.Schema.GetMetadata(MetadataUtils.Kinds.SlotNames, colIndex, ref labels);
+
+            Contracts.Assert(labels.IsDense);
```

Comment:
>Contracts.Assert(labels.IsDense); [](start = 12, length = 33)

A `VBuffer` being sparse isn't an error condition...

Reply:
`.DenseValues` perhaps.

---
In reply to: [190781068](https://github.com/dotnet/machinelearning/pull/239#discussion_r190781068) [](ancestors = 190781068)